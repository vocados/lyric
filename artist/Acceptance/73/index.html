<!doctype html><html lang="ko"><head><script async src="https://www.googletagmanager.com/gtag/js?id=UA-116665513-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-116665513-1');
</script><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link rel="icon" href="/favicon.png"><title>Acceptance - 73 - 가사</title><meta name="description" content="Acceptance - 73 - 노래,음악,가사,노래가사,음악가사"><meta name="og:url" content="http://lyric.vicharas.net/artist/Acceptance/73/"><meta name="og:type" content="blog"><meta name="og:title" content="Acceptance - 73 - 가사"><meta name="og:description" content="Acceptance - 73 - 노래,음악,가사,노래가사,음악가사"><meta name="og:image" content="/favicon.png"><meta name="author" content="lyric"><meta name="twitter:card" content="summary"><meta name="twitter:url" content="http://lyric.vicharas.net/artist/Acceptance/73/"><meta name="twitter:title" content="Acceptance - 73 - 가사"><meta name="twitter:description" content="Acceptance - 73 - 노래,음악,가사,노래가사,음악가사"><meta name="twitter:image" content="/favicon.png"><link rel="canonical" href="http://lyric.vicharas.net/artist/Acceptance/73/"><link href="/static/bootstrap.min.css" rel="stylesheet"><link href="/static/cover.css" rel="stylesheet"><link rel="alternate" href="http://lyric.vicharas.net/artist/Acceptance/73/" hreflang="ko"/></head><body class="text-center"><div class="cover-container d-flex h-100 p-3 mx-auto flex-column"><header class="masthead mb-auto"><div class="inner"><a href="/"><h3 class="masthead-brand">lyric</h3></a></div></header><main role="main" class="inner cover"><h1 class="cover-heading">73</h1><br/><p class="lead"> Artist - Acceptance<br><br> There's nothing to lose, but you're still losing <br>Nothing to give, but you're still giving back <br>There's no one to leave that ain't worth leaving <br>Nothing to love that ain't worth loving back <br><br>And if I ever really want to feel alive <br>I take a look into you when the suns confine <br><br>I'm still here <br>I'm still here <br><br>Should I lose my will <br>Should I chase the thrill <br>I could be anyone <br>Is there a grace to give <br>In the place you live <br>Oh I can't wait anymore <br><br>And if I ever really want to feel alive <br>I take a look into you when the suns confine <br><br>I'm still here <br>You're just in time, in time to save me <br>I'm still here <br>Hell is close, but will you find me <br>Oh when you live in this life alone lose fear <br>I'm still here <br><br>Will you live one more, one more life with me <br>Life with me <br><br>I'm still here <br>You're just in time, in time to save me <br>I'm still here <br>Heaven's close, but will you take me <br>Oh when you live in this life alone lose fear <br>I'm still here <br>I'm still here </p><table class="table table-striped table-dark"><tbody><tr><th scope="row">앨범</th><td>Colliding By Design</td></tr><tr><th scope="row">발매일</th><td>2017-02-24</td></tr><tr><th scope="row">장르</th><td>Rock</td></tr></tbody></table><br><br><div class="list-group" id="listGroup"></div><br><br><div id="disqus_thread"></div><script>
(function() {
var d = document, s = d.createElement('script');
s.src = 'https://lyric-1.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script></main><footer class="mastfoot mt-auto"><div class="inner"><p>since 2018</p></div></footer></div><script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script><script>window.jQuery || document.write('<script src="/static/jquery-slim.min.js"><\/script>')</script><script src="/static/popper.min.js"></script><script src="/static/bootstrap.min.js"></script><script src="/static/post.js"></script></body></html>