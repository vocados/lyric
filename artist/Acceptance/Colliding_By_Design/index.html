<!doctype html><html lang="ko"><head><script async src="https://www.googletagmanager.com/gtag/js?id=UA-116665513-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-116665513-1');
</script><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link rel="icon" href="/favicon.png"><title>Acceptance - Colliding By Design - 가사</title><meta name="description" content="Acceptance - Colliding By Design - 노래,음악,가사,노래가사,음악가사"><meta name="og:url" content="http://lyric.vicharas.net/artist/Acceptance/Colliding_By_Design/"><meta name="og:type" content="blog"><meta name="og:title" content="Acceptance - Colliding By Design - 가사"><meta name="og:description" content="Acceptance - Colliding By Design - 노래,음악,가사,노래가사,음악가사"><meta name="og:image" content="/favicon.png"><meta name="author" content="lyric"><meta name="twitter:card" content="summary"><meta name="twitter:url" content="http://lyric.vicharas.net/artist/Acceptance/Colliding_By_Design/"><meta name="twitter:title" content="Acceptance - Colliding By Design - 가사"><meta name="twitter:description" content="Acceptance - Colliding By Design - 노래,음악,가사,노래가사,음악가사"><meta name="twitter:image" content="/favicon.png"><link rel="canonical" href="http://lyric.vicharas.net/artist/Acceptance/Colliding_By_Design/"><link href="/static/bootstrap.min.css" rel="stylesheet"><link href="/static/cover.css" rel="stylesheet"><link rel="alternate" href="http://lyric.vicharas.net/artist/Acceptance/Colliding_By_Design/" hreflang="ko"/></head><body class="text-center"><div class="cover-container d-flex h-100 p-3 mx-auto flex-column"><header class="masthead mb-auto"><div class="inner"><a href="/"><h3 class="masthead-brand">lyric</h3></a></div></header><main role="main" class="inner cover"><script>
		  (function() {
		    var cx = '013758534719743770872:bre9ch_ozgo';
		    var gcse = document.createElement('script');
		    gcse.type = 'text/javascript';
		    gcse.async = true;
		    gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
		    var s = document.getElementsByTagName('script')[0];
		    s.parentNode.insertBefore(gcse, s);
		  })();
		</script><gcse:search></gcse:search><br/><h1 class="cover-heading">Colliding By Design</h1><br/><p class="lead"> Artist - Acceptance<br><br><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script><ins class="adsbygoogle" style="display:block" data-ad-client="ca-pub-0031623727468371" data-ad-slot="2886566455" data-ad-format="auto"></ins><script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script><br> I begin to look <br>This could only lead to one thing <br>Breathe and bleed so hard it hurts <br>Can't stop, can't stop, colliding by design <br>Tying down my feet to the floor <br>I get hot when I touch your lips <br>Leaving you alone on the shore <br>Sit tight, hold tight <br><br>Night after night <br>Night after night <br><br>Let's kiss before you go away <br>Two burning stars chasing the day <br>There's a look in your eyes you wanna stay <br>So let's kiss before you go away <br><br>I could feel you closing in <br>This could only lead to one thing <br>When I fall, I lose control <br>Can't stop, can't stop, colliding by design <br>Standing here so close to your face <br>I get hot when I touch your lips <br>Everything else seems out of place <br>Sit tight, hold tight <br><br>Night after night <br>Night after night <br><br>Let's kiss before you go away <br>Two burning stars chasing the day <br>There's a look in your eyes you wanna stay <br>So let's kiss before you go away <br><br>Standing here so close to your face <br>Everything else seems out of place <br><br>Let's kiss before you go away <br>Two burning stars chasing the day <br>There's a look in your eyes you wanna stay <br>So let's kiss before you go away </p><table class="table table-striped table-dark"><tbody><tr><th scope="row">앨범</th><td>Colliding By Design</td></tr><tr><th scope="row">발매일</th><td>2017-02-24</td></tr><tr><th scope="row">장르</th><td>Rock</td></tr></tbody></table><br><br><ins class="adsbygoogle" style="display:block" data-ad-client="ca-pub-0031623727468371" data-ad-slot="4233283621" data-ad-format="auto"></ins><script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script><br><div class="list-group" id="listGroup"></div><br><br><div id="disqus_thread"></div><script>
(function() {
var d = document, s = d.createElement('script');
s.src = 'https://lyric-1.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script></main><footer class="mastfoot mt-auto"><div class="inner"><p>since 2018</p></div></footer></div><script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script><script>window.jQuery || document.write('<script src="/static/jquery-slim.min.js"><\/script>')</script><script src="/static/popper.min.js"></script><script src="/static/bootstrap.min.js"></script><script src="/static/post.js"></script></body></html>