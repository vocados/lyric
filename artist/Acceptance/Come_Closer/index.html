<!doctype html><html lang="ko"><head><script async src="https://www.googletagmanager.com/gtag/js?id=UA-116665513-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-116665513-1');
</script><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link rel="icon" href="/favicon.png"><title>Acceptance - Come Closer - 가사</title><meta name="description" content="Acceptance - Come Closer - 노래,음악,가사,노래가사,음악가사"><meta name="og:url" content="http://lyric.vicharas.net/artist/Acceptance/Come_Closer/"><meta name="og:type" content="blog"><meta name="og:title" content="Acceptance - Come Closer - 가사"><meta name="og:description" content="Acceptance - Come Closer - 노래,음악,가사,노래가사,음악가사"><meta name="og:image" content="/favicon.png"><meta name="author" content="lyric"><meta name="twitter:card" content="summary"><meta name="twitter:url" content="http://lyric.vicharas.net/artist/Acceptance/Come_Closer/"><meta name="twitter:title" content="Acceptance - Come Closer - 가사"><meta name="twitter:description" content="Acceptance - Come Closer - 노래,음악,가사,노래가사,음악가사"><meta name="twitter:image" content="/favicon.png"><link rel="canonical" href="http://lyric.vicharas.net/artist/Acceptance/Come_Closer/"><link href="/static/bootstrap.min.css" rel="stylesheet"><link href="/static/cover.css" rel="stylesheet"><link rel="alternate" href="http://lyric.vicharas.net/artist/Acceptance/Come_Closer/" hreflang="ko"/></head><body class="text-center"><div class="cover-container d-flex h-100 p-3 mx-auto flex-column"><header class="masthead mb-auto"><div class="inner"><a href="/"><h3 class="masthead-brand">lyric</h3></a></div></header><main role="main" class="inner cover"><h1 class="cover-heading">Come Closer</h1><br/><p class="lead"> Artist - Acceptance<br><br> She's waited too long <br>For someone to call <br>Head down in the streets <br>Fantasies, what she'll see <br><br>Something's moving the night <br>You'll keep holding me tight <br>She was close <br>For a night some would kill <br>For a night some would kill <br><br>Keep getting closer <br>Keep getting closer <br>Keep getting closer <br>The day calls, the night calls <br>So dreams big, she don't know <br>Keep getting closer <br>The first time's the last time <br><br>Just like the air is breathing <br>She gets this funny feeling <br>Take a moment, take a moment <br>Make the moment oh so long <br><br>Something's moving the night <br>Just keep holding me tight <br>Shade of gold, make it real <br>Dancing souls what you steal <br>Tell me what would you steal <br>Steal, steal, steal <br><br>Keep getting closer <br>Keep getting closer <br>Keep getting closer <br>The day calls, the night calls <br>So dreams big, she don't know <br>Keep getting closer <br>The first time's the last time <br><br>Keep getting closer, closer, closer <br>Keep getting closer, closer, closer <br><br>Keep getting closer <br>Keep getting closer <br>Keep getting closer <br>The day calls, the night calls <br>So dreams big, she don't know <br>Keep getting closer <br>The first time's the last time <br><br>The first time's the last time <br>The first time's the last time <br>The first time's the last time </p><table class="table table-striped table-dark"><tbody><tr><th scope="row">앨범</th><td>Colliding By Design</td></tr><tr><th scope="row">발매일</th><td>2017-02-24</td></tr><tr><th scope="row">장르</th><td>Rock</td></tr></tbody></table><br><br><div class="list-group" id="listGroup"></div><br><br><div id="disqus_thread"></div><script>
(function() {
var d = document, s = d.createElement('script');
s.src = 'https://lyric-1.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script></main><footer class="mastfoot mt-auto"><div class="inner"><p>since 2018</p></div></footer></div><script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script><script>window.jQuery || document.write('<script src="/static/jquery-slim.min.js"><\/script>')</script><script src="/static/popper.min.js"></script><script src="/static/bootstrap.min.js"></script><script src="/static/post.js"></script></body></html>