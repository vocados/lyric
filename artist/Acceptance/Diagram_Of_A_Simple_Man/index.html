<!doctype html><html lang="ko"><head><script async src="https://www.googletagmanager.com/gtag/js?id=UA-116665513-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-116665513-1');
</script><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link rel="icon" href="/favicon.png"><title>Acceptance - Diagram Of A Simple Man - 가사</title><meta name="description" content="Acceptance - Diagram Of A Simple Man - 노래,음악,가사,노래가사,음악가사"><meta name="og:url" content="http://lyric.vicharas.net/artist/Acceptance/Diagram_Of_A_Simple_Man/"><meta name="og:type" content="blog"><meta name="og:title" content="Acceptance - Diagram Of A Simple Man - 가사"><meta name="og:description" content="Acceptance - Diagram Of A Simple Man - 노래,음악,가사,노래가사,음악가사"><meta name="og:image" content="/favicon.png"><meta name="author" content="lyric"><meta name="twitter:card" content="summary"><meta name="twitter:url" content="http://lyric.vicharas.net/artist/Acceptance/Diagram_Of_A_Simple_Man/"><meta name="twitter:title" content="Acceptance - Diagram Of A Simple Man - 가사"><meta name="twitter:description" content="Acceptance - Diagram Of A Simple Man - 노래,음악,가사,노래가사,음악가사"><meta name="twitter:image" content="/favicon.png"><link rel="canonical" href="http://lyric.vicharas.net/artist/Acceptance/Diagram_Of_A_Simple_Man/"><link href="/static/bootstrap.min.css" rel="stylesheet"><link href="/static/cover.css" rel="stylesheet"><link rel="alternate" href="http://lyric.vicharas.net/artist/Acceptance/Diagram_Of_A_Simple_Man/" hreflang="ko"/></head><body class="text-center"><div class="cover-container d-flex h-100 p-3 mx-auto flex-column"><header class="masthead mb-auto"><div class="inner"><a href="/"><h3 class="masthead-brand">lyric</h3></a></div></header><main role="main" class="inner cover"><script>
		  (function() {
		    var cx = '013758534719743770872:bre9ch_ozgo';
		    var gcse = document.createElement('script');
		    gcse.type = 'text/javascript';
		    gcse.async = true;
		    gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
		    var s = document.getElementsByTagName('script')[0];
		    s.parentNode.insertBefore(gcse, s);
		  })();
		</script><gcse:search></gcse:search><br/><h1 class="cover-heading">Diagram Of A Simple Man</h1><br/><p class="lead"> Artist - Acceptance<br><br><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script><ins class="adsbygoogle" style="display:block" data-ad-client="ca-pub-0031623727468371" data-ad-slot="2886566455" data-ad-format="auto"></ins><script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script><br> Put on my nametag <br>Stare in the mirror <br>Give me some answers <br>Make it clearer now <br>And then I'm in my head <br><br>As the sun is getting older <br>While the clock is counting over <br>Am I more than this <br><br>We get caught up between the sea and the sky <br>Give me that look in, that look in your eye <br>We live in black and white <br>We dream in color <br><br>Met a stranger near my station <br>Says you're alright <br>Take a vacation <br>The things to see before I'm dead <br><br>As the air is getting colder <br>While the years are turning over <br>Where will I exist <br><br>We get caught up between the sea and the sky <br>Give me that look in, that look in your eye <br>We live in black and white <br>We dream in color <br><br>We dream in color <br>And we live in black and white <br><br>We get caught up between the sea and the sky <br>We get caught up between the sea and the sky <br>Give me that look in, that look in your eye <br>We live in black and white <br>We dream in color <br><br>We dream, we dream in color <br>And we live in black and white <br>We dream, we dream in color <br>And we live in black and white </p><table class="table table-striped table-dark"><tbody><tr><th scope="row">앨범</th><td>Colliding By Design</td></tr><tr><th scope="row">발매일</th><td>2017-02-24</td></tr><tr><th scope="row">장르</th><td>Rock</td></tr></tbody></table><br><br><ins class="adsbygoogle" style="display:block" data-ad-client="ca-pub-0031623727468371" data-ad-slot="4233283621" data-ad-format="auto"></ins><script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script><br><div class="list-group" id="listGroup"></div><br><br><div id="disqus_thread"></div><script>
(function() {
var d = document, s = d.createElement('script');
s.src = 'https://lyric-1.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script></main><footer class="mastfoot mt-auto"><div class="inner"><p>since 2018</p></div></footer></div><script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script><script>window.jQuery || document.write('<script src="/static/jquery-slim.min.js"><\/script>')</script><script src="/static/popper.min.js"></script><script src="/static/bootstrap.min.js"></script><script src="/static/post.js"></script></body></html>