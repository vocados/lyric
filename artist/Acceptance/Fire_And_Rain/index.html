<!doctype html><html lang="ko"><head><script async src="https://www.googletagmanager.com/gtag/js?id=UA-116665513-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-116665513-1');
</script><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link rel="icon" href="/favicon.png"><title>Acceptance - Fire And Rain - 가사</title><meta name="description" content="Acceptance - Fire And Rain - 노래,음악,가사,노래가사,음악가사"><meta name="og:url" content="http://lyric.vicharas.net/artist/Acceptance/Fire_And_Rain/"><meta name="og:type" content="blog"><meta name="og:title" content="Acceptance - Fire And Rain - 가사"><meta name="og:description" content="Acceptance - Fire And Rain - 노래,음악,가사,노래가사,음악가사"><meta name="og:image" content="/favicon.png"><meta name="author" content="lyric"><meta name="twitter:card" content="summary"><meta name="twitter:url" content="http://lyric.vicharas.net/artist/Acceptance/Fire_And_Rain/"><meta name="twitter:title" content="Acceptance - Fire And Rain - 가사"><meta name="twitter:description" content="Acceptance - Fire And Rain - 노래,음악,가사,노래가사,음악가사"><meta name="twitter:image" content="/favicon.png"><link rel="canonical" href="http://lyric.vicharas.net/artist/Acceptance/Fire_And_Rain/"><link rel="alternate" href="http://lyric.vicharas.net/artist/Acceptance/Fire_And_Rain/" hreflang="ko"/></head><body class="text-center"><div class="cover-container d-flex h-100 p-3 mx-auto flex-column"><header class="masthead mb-auto"><div class="inner"><a href="/"><h3 class="masthead-brand">lyric</h3></a></div></header><main role="main" class="inner cover"><script>
		  (function() {
		    var cx = '013758534719743770872:bre9ch_ozgo';
		    var gcse = document.createElement('script');
		    gcse.type = 'text/javascript';
		    gcse.async = true;
		    gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
		    var s = document.getElementsByTagName('script')[0];
		    s.parentNode.insertBefore(gcse, s);
		  })();
		</script><gcse:search></gcse:search><br/><h1 class="cover-heading">Fire And Rain</h1><br/><p class="lead"> Artist - Acceptance<br><br><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script><ins class="adsbygoogle" style="display:block" data-ad-client="ca-pub-0031623727468371" data-ad-slot="2886566455" data-ad-format="auto"></ins><script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script><br> We came for the fire, but we stay for the rain <br>And we count our time while we're feeling so sane <br>With a cannonball head and a paperback heart <br>Will you see where you've been when you get to the start <br><br>Last season of this masquerade <br>One decision from our lips <br><br>Will you cover your heart when you see what I've done <br>Will you look in my eyes and then stare at the sun <br>I see you falling asleep while I'm saving the wane <br>I hear you calling my name when you make a mistake <br><br>I can see your breath, I'm looking numb now <br>You're running my way, I can't hear a sound <br>With a cannonball crash and a paperback spine <br>If I touch your face, tell me what will I find <br><br>Last season of this masquerade <br>One decision from our lips <br><br>Will you cover your heart when you see what I've done <br>Will you look in my eyes and then stare at the sun <br>I see you falling asleep while I'm saving the wane <br>I hear you calling my name when you make a mistake <br><br>Let me down by the foolish flame <br>And I'll take the blame <br>I'll take the blame <br><br>We came for the fire and drowned in the rain <br>We went in alone and came out the same <br><br>Will you cover your heart when you see what I've done <br>Will you look in my eyes and then stare at the sun <br>I see you falling asleep while I'm saving the wane <br>I hear you calling my name when you make a mistake </p><table class="table table-striped table-dark"><tbody><tr><th scope="row">앨범</th><td>Colliding By Design</td></tr><tr><th scope="row">발매일</th><td>2017-02-24</td></tr><tr><th scope="row">장르</th><td>Rock</td></tr></tbody></table><br><br><ins class="adsbygoogle" style="display:block" data-ad-client="ca-pub-0031623727468371" data-ad-slot="4233283621" data-ad-format="auto"></ins><script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script><br><div class="list-group" id="listGroup"></div><br><br><div id="disqus_thread"></div><script>
(function() {
var d = document, s = d.createElement('script');
s.src = 'https://lyric-1.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script></main><footer class="mastfoot mt-auto"><div class="inner"><p>since 2018</p></div></footer></div><script src="/static/post.js"></script></body></html>