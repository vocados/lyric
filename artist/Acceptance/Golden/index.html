<!doctype html><html lang="ko"><head><script async src="https://www.googletagmanager.com/gtag/js?id=UA-116665513-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-116665513-1');
</script><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link rel="icon" href="/favicon.png"><title>Acceptance - Golden - 가사</title><meta name="description" content="Acceptance - Golden - 노래,음악,가사,노래가사,음악가사"><meta name="og:url" content="http://lyric.vicharas.net/artist/Acceptance/Golden/"><meta name="og:type" content="blog"><meta name="og:title" content="Acceptance - Golden - 가사"><meta name="og:description" content="Acceptance - Golden - 노래,음악,가사,노래가사,음악가사"><meta name="og:image" content="/favicon.png"><meta name="author" content="lyric"><meta name="twitter:card" content="summary"><meta name="twitter:url" content="http://lyric.vicharas.net/artist/Acceptance/Golden/"><meta name="twitter:title" content="Acceptance - Golden - 가사"><meta name="twitter:description" content="Acceptance - Golden - 노래,음악,가사,노래가사,음악가사"><meta name="twitter:image" content="/favicon.png"><link rel="canonical" href="http://lyric.vicharas.net/artist/Acceptance/Golden/"><link href="/static/bootstrap.min.css" rel="stylesheet"><link href="/static/cover.css" rel="stylesheet"><link rel="alternate" href="http://lyric.vicharas.net/artist/Acceptance/Golden/" hreflang="ko"/></head><body class="text-center"><div class="cover-container d-flex h-100 p-3 mx-auto flex-column"><header class="masthead mb-auto"><div class="inner"><a href="/"><h3 class="masthead-brand">lyric</h3></a></div></header><main role="main" class="inner cover"><h1 class="cover-heading">Golden</h1><br/><p class="lead"> Artist - Acceptance<br><br> We're all just chemicals <br>We're all just chemicals <br>You can lose yourself in me <br>And I feel it when you breathe <br>We're all just physical <br>And so subliminal <br>I can lose myself in you <br>And you pull, you pull me through <br><br>So I'm told, we grow old and move on, and move on <br>So I'm told, to fear gold, and shiver, shiver <br>As you hold me close and take me through the night <br>The two of us are bound to get the ride <br>We're getting high <br><br>And we're all just chemicals <br>And we're all just chemicals <br>You can lose yourself in me <br>And I feel you when you breathe <br>We're all just physical <br>And you're so subliminal <br>And I lose myself in you <br>And you pull, you pull me through <br><br>Check my eyes sharing lies with my head, with my head <br>I am told, to fear gold, shiver, shiver <br>As you hold me close and take me through the night <br>The two of us are bound to get the ride <br>We're getting high <br><br>And we're all just chemicals <br>We're all just chemicals <br>Lose myself <br><br>It gets better, better than this <br>It gets better, better than this <br>So golden, golden <br>It gets better, better than this <br>It gets better, better than this <br>So golden, golden <br>It gets better, better than this <br>It gets better, better than this <br>So golden, golden <br>It gets better, better than this <br>It gets better, better than this <br>So golden, golden <br><br>It gets better, better than this <br>It gets better, better than this <br>So golden, golden </p><table class="table table-striped table-dark"><tbody><tr><th scope="row">앨범</th><td>Colliding By Design</td></tr><tr><th scope="row">발매일</th><td>2017-02-24</td></tr><tr><th scope="row">장르</th><td>Rock</td></tr></tbody></table><br><br><div class="list-group" id="listGroup"></div><br><br><div id="disqus_thread"></div><script>
(function() {
var d = document, s = d.createElement('script');
s.src = 'https://lyric-1.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script></main><footer class="mastfoot mt-auto"><div class="inner"><p>since 2018</p></div></footer></div><script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script><script>window.jQuery || document.write('<script src="/static/jquery-slim.min.js"><\/script>')</script><script src="/static/popper.min.js"></script><script src="/static/bootstrap.min.js"></script><script src="/static/post.js"></script></body></html>